testcases/bestest_hydronic_heat_pump/models/generate_data.py

```diff
@@ -155,5 +155,53 @@
 # Add CO2 limits
 df['UpperCO2[1]'] = 894.
 
+def hold_step_values_in_event_times(df, var, event_freq):
+    ''' This method goes through the index of a data frame to inspect
+    when step changes take place for a variable and makes sure that
```

Comment:
Maybe just clarify as "positive" step changes?

Reply:
And here: https://github.com/ibpsa/project1-boptest/pull/311/commits/184e08b3b3dbe426adf791460ebc01f9cadc2569.